cmd/juju/commands/common.go

```diff
@@ -148,24 +148,24 @@ func resolveCharmURL(curlStr string, csParams charmrepo.NewCharmStoreParams, rep
 		// The URL is already fully resolved; do not
 		// bother with an unnecessary round-trip to the
 		// charm store.
-		curl, err := ref.URL("")
+		url, err := ref.URL("")
```

Comment:
I don't mind it called curl actually. It means that it doesn't alias with the net/url package. It's still a charm.URL (and there are other instances of the curl name in this package too).